src/libtiled/tile.h

```diff
@@ -116,6 +116,9 @@ class TILEDSHARED_EXPORT Tile : public Object
 
     QPoint offset() const;
 
+    QPoint tileOffsetXY() const;
+    void setTileOffsetXY(QPoint offset);
```

Comment:
Please rename `Tile::offset` to `Tile::totalOffset` and then rename these functions to `Tile::offset` and `Tile::setOffset`. Then, if you make `Tile::totalOffset` return the sum of the local tile offset and the tileset tile offset, you would no longer need to change `maprenderer.cpp` and I think it may resolve some issues with dealing with the offset for tile objects.